packages/react/src/patterns/sub-patterns/Card/Card.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { ClickableTile, Tile } from 'carbon-components-react';
+import {
+  settings as ddsSettings,
+  markdownToHtml,
+} from '@carbon/ibmdotcom-utilities';
+import classNames from 'classnames';
+import { CTA } from '../../../components/CTA';
+import { Image } from '../../../components/Image';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { settings } from 'carbon-components';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * Card Link Component
+ *
+ * @param {object} props react proptypes
+ * @returns {object} JSX object
+ */
+const Card = ({ title, copy, cta, image, className, type, ...props }) => {
+  const CardTile = type === 'link' ? ClickableTile : Tile;
+  return (
+    <CardTile
+      data-autoid={`${stablePrefix}--card`}
+      className={classNames(`${prefix}--card`, className)}
+      href={cta.href}
```

Comment:
Does this cause any problem staying here if only `Tile` is used?

Reply:
@photodow no, it shouldn't.

Reply:
yeah it's fine